pkg/scheduler/api/job_info.go

```diff
@@ -116,8 +116,7 @@ func getTaskID(pod *v1.Pod) TaskID {
 
 // NewTaskInfo creates new taskInfo object for a Pod
 func NewTaskInfo(pod *v1.Pod) *TaskInfo {
-	req := GetPodResourceRequest(pod)
-	initResreq := GetPodResourceRequest(pod)
+	resReq := GetPodResourceRequest(pod)
```

Comment:
I think we should distinguish initReq vs. req.

Reply:
changed to 
```go
	initResReq := GetPodResourceRequest(pod)
	resReq := initResReq
```